src/app/components/drawer/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC } from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { Box, Text } from '@gluestack-ui/themed';
+import { DrawerContentComponentProps } from '@react-navigation/drawer';
+import { UserAvatar } from '@src/assets/svg/avatar/user-avatar';
+import {
+  HomeIcon,
+  LogoutIcon,
+  ProfileIcon,
+  SettingsIcon,
+} from '@src/assets/svg/navigation-icons/';
+
+const ROUTES = {
+  HOME: 'HomeScreen',
+  PROFILE: 'Profile',
+  SETTINGS: 'SettingsScreen',
+};
```

Comment:
These routes may we could moved inside the constants directory inside a file called routes. WDYT?